cypress-shared/support/shopper-location/common.js

```diff
@@ -126,3 +126,25 @@ export function orderProductTestCase(prefix, data) {
     cy.orderProduct()
   })
 }
+
+export function verifyHomePage(city, postalCode, distance = false) {
```

Comment:
This is added in shopper location 
Can you please move to location availability

Reply:
You forgot to remove this